src/utils.cpp

```diff
@@ -244,25 +244,9 @@ std::vector<std::string> utils::tokenize_spaced(const std::string& str,
 }
 
 std::string utils::consolidate_whitespace(const std::string& str,
-	std::string whitespace)
-{
-	std::string result;
-	std::string::size_type last_pos = str.find_first_not_of(whitespace);
-	std::string::size_type pos = str.find_first_of(whitespace, last_pos);
+	__attribute__((unused))std::string whitespace) {
```

Comment:
Please make a separate commit that removes that argument altogether. If it's not used, it's a dead weight.